leaf-ui/rappid-extensions/ConfigInspector.js

```diff
@@ -347,7 +347,13 @@ var ConfigInspector = Backbone.View.extend({
 
     /** Create and add a new config model to the collection */
     addNewConfig: function () {
-        var configModel = new ConfigBBM({ name: "Request " + (this.collection.length + 1), results: new ResultCollection([]) })
+        var configModel; 
+        if (this.collection.length < 1){
+                configModel = new ConfigBBM({ name: "Result " + 1, results: new ResultCollection([]) });
+        } else { 
+            var nameIndex = parseInt(this.collection.at(this.collection.length - 1 ).get('name').split(' ').pop()) + 1;
```

Comment:
When a request is renamed to something not containing a number, this replaces the last word in the name with `NaN`. This also returns odd results when a Request is named just a number or has an empty name. (Also, less importantly, if we rename a Request to a different number, we start counting from that number even if that isn't the total number of Requests ever generated - not sure whether we like this behavior.)

<img width="204" alt="Screen Shot 2021-09-18 at 13 34 47" src="https://user-images.githubusercontent.com/59173829/133897526-d4eef968-95d4-4a5e-b31c-c0e9aea3101d.png">